mods/ra/mod.yaml

```diff
@@ -67,8 +67,6 @@ Chrome:
 Assemblies:
 	./mods/common/OpenRA.Mods.Common.dll
 	./mods/ra/OpenRA.Mods.RA.dll
-	./mods/cnc/OpenRA.Mods.Cnc.dll
```

Comment:
This will break the custom RA maps that port the Ion Cannon and other TD things into RA.